sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -180,21 +184,45 @@ cdef class StateSampler(object):
         self.scoped_states_by_index[self.current_state_index].name,
         self.state_transition_count)
 
-  def scoped_state(self, name):
-    """Returns a context manager managing transitions for a given state."""
-    cdef ScopedState scoped_state = self.scoped_states_by_name.get(name, None)
+  # TODO(pabloem) - Make state_name required once all callers migrate,
```

Comment:
<!--new_thread; commit:f10b3747bb7c869dab00ffc28d50b4cab4e238aa; resolved:0-->
nit: Can you change this to follow the project style of `# TODO(XXX): YYY`?